src/gc/config.d

```diff
@@ -20,12 +20,13 @@ extern extern(C) __gshared bool rt_envvars_enabled;
 extern extern(C) __gshared bool rt_cmdline_enabled;
 extern extern(C) __gshared string[] rt_options;
 
+__gshared Config config;
+
 struct Config
-{
+{   bool initialized;
```

Comment:
Whoops! Nope, this isn't used anywhere. I had intended to test something with this, but forgot to double check the diff before I pushed and it wasn't deleted.